frontend/src/js/components/settings/role-management/DeleteRoleDialog.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2024 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+import { FunctionComponent } from 'react';
+
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle } from '@mui/material';
+
+interface Props {
+  dismiss: () => void;
+  submit: () => void;
+  name: string;
+  open: boolean;
+}
+
+export const DeleteRoleDialog: FunctionComponent<Props> = ({ dismiss, open, submit, name }) => (
```

Comment:
Is it possible to use our `ConfirmModal` with some adjustments?

Reply:
Good point! 🙊 if we don't tell anyone I think it should be fine 😄